src/core/create_compilers/RollupCompiler.ts

```diff
@@ -154,11 +156,51 @@ export default class RollupCompiler {
 
 		const that = this;
 
-		// TODO this is hacky. refactor out into an external rollup plugin
-		(mod.plugins || (mod.plugins = [])).push(css_chunks({ injectImports: true }));
-		if (!/[\\/]client\./.test(entry_point)) {
-			return mod;
-		}
+		/**
+		 * Track css code hashes for proper chunk hashing
+		 */
+		const css_hashing = () => {
+			const css_hashes: Record<string, string> = {};
+			return <Plugin>{
+				name: 'sapper-css-hashing',
+				augmentChunkHash(this: PluginContext, chunk: PreRenderedChunk) {
+					let hash: Hash;
+					const module_files = Object.keys(chunk.modules).map(module_id => {
+						/**
+						 * Save `module_file` which represents the id without file extension. This way it's easy to determine
```

Comment:
I'd probably just use `//` style comments for variables and reserve the other style for method declarations